src/main/asciidoc/en/Ch04_AsyncJobWithWebContainer.adoc

```diff
@@ -434,6 +434,21 @@ Procedure will be same even when conventional Web application (Servlet/JSP) or S
 .Bean definition file to be added/deleted from a blank project
 image::Ch04_AsyncJobWithWebContainer_AppendBeanDefinitionsOnBlank.png[AppendBeanDefinitionsOnBlank]
 
+[cols="10,90", options="header"]
+.Bean definition file to be added/deleted
+|===
+
+|S. No.
+|Description
+
+|(1)
+|(2) is created in spring-mvc.xml, delete it as it is not required.
```

Comment:
@bttendulkara L-445
Wrong translation --> (2) is created in spring-mvc.xml, delete it as it is not required.
Correct translation --> spring-mvc.xml created in (2). Delete this as it is not required.